tests/test_service.py

```diff
@@ -75,6 +75,33 @@ def test_index(self):
         self.assertEqual(resp.status_code, status.HTTP_200_OK)
         self.assertIsNotNone(data)
 
+    def test_get_related_products(self):
+        """ Get related products by id tests"""
+        # Test for valid id
+        recommendation = self._create_recommendations(count=20, by_status=True)
+    
+        resp = self.app.get\
+            ("/recommendations/" + str(recommendation[0][0].id))
+
+        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        self.assertEqual(isinstance(resp, object), True, "Received incorrect recommendation")
```

Comment:
Can you check if the returned resp.get_json() has recommendation objects that have id == recommendation[0][0].id?